app/models/importers/all_taxons.rb

```diff
@@ -0,0 +1,17 @@
+module Importers
+  class AllTaxons
+    attr_accessor :taxonomies_service
+
+    def initialize
+      @taxonomies_service = TaxonomiesService.new
+    end
+
+    def run
+      taxonomies_service.find_each do |attributes|
+        content_id = attributes.fetch(:content_id)
+        taxonomy = Taxonomy.find_or_create_by(content_id: content_id)
```

Comment:
Left as is as per our conversation - will wait to refactor if the importer gets more complex.